snowpack/src/build/build-pipeline.ts

```diff
@@ -84,6 +89,9 @@ async function runPipelineLoadStep(
 
           // if source maps disabled, don’t return any
           if (!sourceMaps) result[ext].map = undefined;
+
+          // clean up empty files
+          if (!result[ext].code) delete result[ext];
```

Comment:
@FredKSchott This, however, is a possibly-controversial change. This asks the question: _should we rely on every plugin to clean up perfectly after itself? Or should Snowpack assume an empty file shouldn’t be generated?_

If the latter, then maybe this is a good change to make? I don’t see this causing any issues, unless in some cases an empty file isn’t a mistake (though I can’t think of any of those scenarios off the top of my head)

Reply:
I thought that today we treated empty string as equivalent to a file that shouldn't be built. I think i saw this issue pop up in some other work that I was doing, so maybe we need to revisit this specifically. it's also different for a plugin that creates more than one file.

Reply:
Yeah so we did check that the plugin generated something, but we weren’t checking if it returned `{ '.js': { code: '' } }`. This is the change here.